auxiliary/stata-guide.tex

```diff
@@ -21,61 +21,27 @@
 We hope that this guide can help increase the emphasis
 given to using, improving, sharing, and standardizing code style among the Stata community.
-Style guides are the most important tool in how you, like an architect,
+We believe these resources can help anyone write more understandable code,
+and how you, like an architect,
 can draw a blueprint that can be understood and used by everyone in your trade.
 
 \end{fullwidth}
 
 %------------------------------------------------
 
-\section{Using the code examples in this book}
-
-In the book, code examples are presented like the following:
-
-\codeexample{code.do}{./code/code.do}
-
-We ensure that each code block runs independently, is well-formatted,
-and uses built-in functions as much as possible.
-We will point to user-written functions when they provide important tools.
-In particular, we point to two suites of Stata commands developed by DIME Analytics,
-\texttt{ietoolkit}\sidenote{\url{https://dimewiki.worldbank.org/ietoolkit}} and
-\texttt{iefieldkit},\sidenote{\url{https://dimewiki.worldbank.org/iefieldkit}}
-which standardize our core data collection, management, and analysis workflows.
-We will comment the code generously (as you should),
-but you should reference Stata help-files by writing \texttt{help [command]}
-whenever you do not understand the command that is being used.
-We hope that these snippets will provide a foundation for your code style.
-Providing some standardization to Stata code style is also a goal of this team.
-
-You can access the raw code used in examples in this book in several ways.
-We use GitHub to version control everything in this book, the code included.
-To see the code on GitHub, go to: \url{https://github.com/worldbank/d4di/tree/master/code}.
-If you are familiar with GitHub you can fork the repository and clone your fork.
-We only use Stata's built-in datasets in our code examples,
-so you do not need to download any data.
-If you have Stata installed on your computer, then you will already have the data files used in the code.
-
-A less technical way to access the code is to click the individual file in the URL above, then click
-the button that says \textbf{Raw}. You will then get to a page that looks like the one at:
-\url{https://raw.githubusercontent.com/worldbank/d4di/master/code/code.do}.
-There, you can copy the code from your browser window to your do-file editor with the formatting intact.
-This method is only practical for a single file at the time.
-If you want to download all code used in this book, you can do that at:
-\url{https://github.com/worldbank/d4di/archive/master.zip}. That link offers a \texttt{.zip} file download
-with all the content used in writing this book, including the \LaTeX{} code used for the book itself. After
-extracting the .zip-file you will find all the code in a folder called \texttt{/code/}.
-
-\subsection{Writing and Understanding Stata code}
+\section{Writing code that is both correct and good}
```

Comment:
```suggestion
\section{Writing good code}
```

It's odd to say both here in the title and then the first line says "good code is correct" I think